src/test/java/de/tum/in/www1/artemis/service/FileServiceTest.java

```diff
@@ -298,4 +298,17 @@ void testDeleteFiles_shouldNotThrowException() {
         Path path = Path.of("some-random-path-which-does-not-exist");
         assertDoesNotThrow(() -> fileService.deleteFiles(List.of(path)));
     }
+
+    @Test
+    void testRenameDirectory() {
+        // File is deleted automatically after test
```

Comment:
the test should also not use toFile.
and where is the file deleted again?
we should probably also assert that the the path of source dir equals the one of target dir after the rename.

Reply:
The test class features a @AfterEach function which deletes all files `./exportTest`. Which is where the generated files are located. So everything is taken care of.

Reply:
Why should the path of the sourceDir equal the targetDir?

Reply:
that's what I would expect from the name. It renames the sourceDir to targetDir.